libs/libc/pthread/pthread_exit.c

```diff
@@ -55,6 +55,10 @@
 
 void pthread_exit(FAR void *exit_value)
 {
+  FAR struct tls_info_s *info = tls_get_info();
+
```

Comment:
SHould we also move checking the flag here from nxsig_abnormal_termination or is doing it there enough?

Reply:
hi, pussuw, i think checking in nxsig_abnormal_termination is enough, because pthread_exit can only be called repeatedly in nxsig_abnormal_termination.